src/Analysis/GWASUIApp/GWASUIApp.jsx

```diff
@@ -0,0 +1,482 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Steps, Button, Space, Table, Input, Form, Result, Alert, Popconfirm, InputNumber, Select,
+} from 'antd';
+import { userHasMethodForServiceOnResource } from '../../authMappingUtils';
+import GWASUIJobStatusList from './GWASUIJobStatusList';
+import './GWASUIApp.css';
+import { getAllSources, getCohortJsonByName } from '../../cohortMiddleware';
+
+const { Step } = Steps;
+
+const steps = [
+  {
+    title: 'Step 1',
+    description: 'Select a cohort for GWAS',
+  },
+  {
+    title: 'Step 2',
+    description: 'Select harmonized variables for phenotypes and covariates',
+  },
+  {
+    title: 'Step 3',
+    description: 'Select which variable is your phenotype and remove unwanted covariates',
+  },
+  {
+    title: 'Step 4',
+    description: 'Set workflow parameters',
+  },
+  {
+    title: 'Step 5',
+    description: 'Submit GWAS job',
+  },
+];
+
+const GWASUIApp = (props) => {
+  const [form] = Form.useForm();
+  const [current, setCurrent] = useState(0);
+  const [selectedCohort, setSelectedCohort] = useState(undefined);
+  const [selectedConcepts, setSelectedConcepts] = useState([]);
+  const [numOfPC, setNumOfPC] = useState(3);
+  const [selectedPhenotype, setSelectedPhenotype] = useState(undefined);
+  const [selectedCovariates, setSelectedCovariates] = useState([]);
+  const [jobName, setJobName] = useState(undefined);
+  const [showJobSubmissionResult, setShowJobSubmissionResult] = useState(false);
+  const [jobSubmittedRunID, setJobSubmittedRunID] = useState(undefined);
+  const [marinerJobStatus, setMarinerJobStatus] = useState([
+    {
+      runID: 'run-10000',
+      status: 'completed',
+      output: [
+        {
+          name: 'test_null_model_report.html',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_null_model_report.html',
+        },
+        {
+          name: 'test_null_model_invnorm_report.html',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_null_model_invnorm_report.html',
+        },
+        {
+          name: 'test_single_assoc_qq.png',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_single_assoc_qq.png',
+        },
+        {
+          name: 'test_single_assoc_manhattan.png',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_single_assoc_manhattan.png',
```

Comment:
be aware these dummy results are hardcoded to be pulled from qa-mikcy

if you deply this to other env the behavior might get funky